Compile error with godeltaprof@v0.1.6

<!--
Please use this template while reporting a bug and provide as much info as possible. Not doing so may result in your bug not being addressed in a timely manner. Thanks!
-->

**What happened**:
Error during compilation:
```
[root@master juicefs]# make
go version
go version go1.23.2 (Red Hat 1.23.2-1.el9) linux/amd64
go build -ldflags="-s -w -X github.com/juicedata/juicefs/pkg/version.revision=cd871d19 -X github.com/juicedata/juicefs/pkg/version.revisionDate=2024-08-30"  -o juicefs .
# github.com/grafana/pyroscope-go/godeltaprof/internal/pprof
../go/pkg/mod/github.com/grafana/pyroscope-go/godeltaprof@v0.1.6/internal/pprof/delta_mutex.go:30:20: undefined: runtime_cyclesPerSecond
../go/pkg/mod/github.com/grafana/pyroscope-go/godeltaprof@v0.1.6/internal/pprof/proto.go:320:8: undefined: runtime_expandFinalInlineFrame
make: *** [Makefile:24: juicefs] Error 1
```
**Solution**:
Bump up the version of godeltaprof in go.mod to 0.1.8

**Environment**:
- JuiceFS version (use `juicefs --version`): 1.2.1+2024-08-30.cd871d19
- OS (e.g `cat /etc/os-release`): CentOS Stream 9
- Kernel (e.g. `uname -a`): 5.14.0-522.el9.x86_64

--- a/go.mod
+++ b/go.mod
@@ -151,7 +151,7 @@ require (
 	github.com/gopherjs/gopherjs v0.0.0-20181017120253-0766667cb4d1 // indirect
 	github.com/gorilla/handlers v1.5.1 // indirect
 	github.com/gorilla/mux v1.8.0 // indirect
-	github.com/grafana/pyroscope-go/godeltaprof v0.1.6
+	github.com/grafana/pyroscope-go/godeltaprof v0.1.8
 	github.com/grpc-ecosystem/go-grpc-middleware v1.1.0 // indirect
 	github.com/hashicorp/go-cleanhttp v0.5.1 // indirect
 	github.com/hashicorp/go-immutable-radix v1.3.0 // indirect
@@ -169,7 +169,7 @@ require (
 	github.com/jcmturner/rpc/v2 v2.0.3 // indirect
 	github.com/jmespath/go-jmespath v0.4.0 // indirect
 	github.com/jtolds/gls v4.20.0+incompatible // indirect
-	github.com/klauspost/compress v1.17.7 // indirect
+	github.com/klauspost/compress v1.17.8 // indirect
 	github.com/klauspost/cpuid v1.3.1 // indirect
 	github.com/klauspost/cpuid/v2 v2.2.7 // indirect
 	github.com/klauspost/pgzip v1.2.5 // indirect

